app/views/group/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<% dev = @group.first %>
+<h1>Group Information for <%= "#{dev.platform}-#{dev.instance_type || "Group#{dev.group}"}-#{dev.location}" %></h1>
```

Comment:
Could we replace spaces in `dev.platform` with dashes? It looks a little inconsistent in Alces Cloud group names to have a single space amongst the dashes